dashboard/app/models/stage.rb

```diff
@@ -106,8 +108,8 @@ def summarize
         flex_category: localized_category,
         lockable: !!lockable,
         levels: cached_script_levels.map(&:summarize),
-        description_student: (I18n.t "data.script.name.#{script.name}.stages.#{name}.description_student", default: ''),
-        description_teacher: (I18n.t "data.script.name.#{script.name}.stages.#{name}.description_teacher", default: '')
+        description_student: render_restricted_markdown(I18n.t("data.script.name.#{script.name}.stages.#{name}.description_student"), default: ''),
+        description_teacher: render_restricted_markdown(I18n.t("data.script.name.#{script.name}.stages.#{name}.description_teacher"), default: '')
```

Comment:
nice catch. i originally didnt have parens, then the linter complained at me, so i added them (in the wrong spot) :man_facepalming: